pkg/config/protocol/instance.go

```diff
@@ -54,6 +54,8 @@ const (
 	Redis Instance = "Redis"
 	// MySQL declares that the port carries MySQL traffic.
 	MySQL Instance = "MySQL"
+	// HBONE declares that the port carries HBONE traffic.
+	HBONE Instance = "HBONE"
```

Comment:
Don't we have a dedicated port ?

Reply:
Or 2 ports actually - one for HBone and one for 'plain text/trusted network' hbone, i.e. h2 or h2c

Reply:
This is not something a user can configure by portname on Service -- but used for internal. Not by convention, this is enforced.